src/compiler/output-targets/output-utils.ts

```diff
@@ -14,36 +14,6 @@ export const relativeImport = (pathFrom: string, pathTo: string, ext?: string, a
   return normalizePath(`${relativePath}/${basename(pathTo, ext)}`);
 };
 
-export const getDistEsmDir = (outputTarget: d.OutputTargetDist, sourceTarget?: d.SourceTarget) =>
```

Comment:
I couldn't find any uses of these helper functions, and builds and tests pass without them so I believe it is safe to delete them.